packages/cli/src/commands/move/tasks/initialize.ts

```diff
@@ -18,7 +15,11 @@ export function initTask(
       ctx.childPackage = options.childPackage;
       // source file or dir should always exist within the basePath
       if (options.source) {
-        [ctx.jsSourcesAbs, ctx.jsSourcesRel] = validateSourcePath(options.basePath, options.source);
+        [ctx.jsSourcesAbs, ctx.jsSourcesRel] = validateSourcePath(
+          options.basePath,
+          options.source,
+          'js'
```

Comment:
Can't you have `gjs` files as well?

Reply:
yes. the `js` is just an identifier which will return all `*js` (right now its gjs, js)

Reply:
ill change the last arg for `validateSourcePath` to be either a glob or maybe array of extensions rather than obfuscate via
```
  const globPaths = fileType === 'js' ? ['.js,.gjs'] : ['.ts,.gts'];
```